src/components/allReviews/allReviews.jsx

```diff
@@ -0,0 +1,57 @@
+import axios from "axios"
+import { useState, useEffect } from "react"
+import { useParams } from 'react-router-dom'
+import AllReviewsCard from "./allReviewsCard"
+
+
+const AllReviews = () => {
+    const params = useParams()
+    const [loading, setLoading] = useState(true)
+    const [item, setItem] = useState([])
```

Comment:
naming of item/setItem in useState is a bit misleading here, implies that it is a single object but we are dealing with a response array of review objects, I'd suggest renaming to be more semantic here